virttest/storage.py

```diff
@@ -297,7 +297,11 @@ def cmp(x, y):
                                         "selector = '%s'" %
                                         (re_name, matching_images,
                                          indirect_image_select))
-        for protected in params.get('indirect_image_blacklist', '').split(' '):
+        indirect_image_blacklist = params.get('indirect_image_blacklist', '').split(' ')
+        for disk in params.get('indirect_image_blacklist', '').split(' '):
+            if re.search(disk, image_name):
+                indirect_image_blacklist.remove(disk)
```

Comment:
I think that we could resolve this issue at the case level since I have confirmed the blacklist then we can modify or update the `indirect_image_blacklist` directly in the cfg or py.